TeamCode/src/main/java/org/firstinspires/ftc/teamcode/api/Drone.kt

```diff
@@ -0,0 +1,25 @@
+package org.firstinspires.ftc.teamcode.api
+
+import com.qualcomm.robotcore.eventloop.opmode.OpMode
+import com.qualcomm.robotcore.hardware.Servo
+import org.firstinspires.ftc.teamcode.utils.RobotConfig
+
+/**
+ * An API to control the drone launcher
+ */
+object Drone: API() {
+    private lateinit var pin: Servo
+
+    override fun init(opMode: OpMode) {
+        super.init(opMode)
+
+        this.pin = this.hardwareMap.get(Servo::class.java, "pin")
+    }
+
+    /** Moves the pin to the open position*/
+    fun releasePin() {
+        this.pin.position = RobotConfig.Drone.OPEN_PIN
+    }
+
+
```

Comment:
It may be useful to also have a `resetPin` function that puts the pin back in it's original location. It could even been called in the `init` function to ensure it's in the right place.